### PR TITLE
[RFR] adjust Intellij file path

### PR DESCRIPTION
**Pull request by Neilhamza**
when installing IntelliJ - the executable file is saved in bin folder
the version details is detailed in product-info.json file (this is for all jetBrains apps)
adjusted code to match these requirements

### CODE REVIEW

File: src/models/IDE/Intellij.py [hunk 1]
```diff
@@ -23,8 +24,14 @@ def __init__(self):
         self.configurations = []
 
     def get_ide_version(self, ide_directory):
-        pattern = re.compile(r"\d{3}\.\d{4}\.\d{2}")
-        return [name for name in os.listdir(ide_directory) if pattern.search(name) is not None][0]
+        info_file_path = os.path.join(ide_directory, 'product-info.json')
+        try:
+            with open(info_file_path, 'r') as info_file:
```
Comment (midays) [inline src/models/IDE/Intellij.py]: You can use the `read_file` function in the util folder and parse the content

File: src/models/IDE/Intellij.py [hunk 2]
```diff
@@ -23,8 +24,14 @@ def __init__(self):
...
+                info_data = json.load(info_file)
+                return info_data["version"]
+        except FileNotFoundError:
```
Comment (midays) [inline src/models/IDE/Intellij.py]: This `try, except` block can be added to the read_file method

**Review by midays — CHANGES_REQUESTED** [review]

File: src/models/IDE/Intellij.py (hunk 1 above)
Comment (Neilhamza) [reply src/models/IDE/Intellij.py]: done

File: src/models/IDE/Intellij.py (hunk 2 above)
Comment (Neilhamza) [reply src/models/IDE/Intellij.py]: done

**Review by sshveta — APPROVED** [review]

**Review by midays — APPROVED** [review]

**Pull request merged**